components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -799,9 +799,25 @@ function ImageBuildView(props: ImageBuildViewProps) {
                 <WorkspaceLogs logsEmitter={logsEmitter} errorMessage={props.error?.message} />
             </Suspense>
             {!!props.onStartWithDefaultImage && (
-                <button className="mt-6 secondary" onClick={props.onStartWithDefaultImage}>
-                    Continue with Default Image
-                </button>
+                <>
+                    <div className="mt-6 w-11/12 lg:w-3/5">
+                        <p className="text-center text-gray-400 dark:text-gray-500">
+                            <span className="font-bold">Did you know?</span> You can use <code>gp rebuild</code> to
+                            validate and debug your custom workspace image before pushing &nbsp;&middot;&nbsp;
```

Comment:
**thought**: +1 for this minimal addition here. We could introduce an info variant[[1](https://github.com/gitpod-io/gitpod/blob/5d38b2d2209f3d1b68b1fd60edbb3c621068960a/components/dashboard/src/components/Alert.tsx#L20)] or introduce a muted variant (see screenshot below) that can be also center aligned for the alert component, but going with a plain text for now should suffice and looks balanced with the logs above.

<img width="1504" alt="Frame 1084" src="https://user-images.githubusercontent.com/120486/212888181-e2f2f898-bd64-4e13-8970-ea8e4d8b6bdc.png">